GUIDE.md

```diff
@@ -223,29 +240,70 @@ This can be done by adding a `*` to the write access array:
 ```javascript
 const ipfs = new IPFS()
 ipfs.on('ready', async () => {
-  const orbitdb = new OrbitDB(ipfs)
+  const orbitdb = await OrbitDB.createInstance(ipfs)
 
-  const access = {
+  const options = {
     // Give write access to everyone
-    write: ['*'],
+    accessController: {
+      write: ['*']
+    }
   }
 
-  const db = await orbitdb.keyvalue('first-database', access)
+  const db = await orbitdb.keyvalue('first-database', options)
   console.log(db.address.toString())
   // /orbitdb/QmRrauSxaAvNjpZcm2Cq6y9DcrH8wQQWGjtokF4tgCUxGP/first-database
 })
 ```
 
 Note how the access controller hash is different compared to the previous example!
 
+#### Granting access after database creation
+
+To give access to another peer after the database has been created, you must set the access-controller `type` to an `AccessController` which supports dynamically adding keys such as `OrbitDBAccessController`.
```

Comment:
We could add a small clarification in *"...which supports dynamically adding keys such..."* that *"...which supports dynamically adding* ***write access*** *such..."*